pkg/userdata/centos/userdata.go

```diff
@@ -138,6 +144,7 @@ func (p Provider) UserData(spec machinesv1alpha1.MachineSpec, kubeconfig *client
 		ClusterDNSIPs:     clusterDNSIPs,
 		KubeadmCACertHash: kubeadmCACertHash,
 		ServerAddr:        serverAddr,
+		IsKube18:          isKube18,
```

Comment:
Sprig has the function `semverCompare` http://masterminds.github.io/sprig/semver.html